pkg/debug/spec.sh

```diff
@@ -478,7 +478,8 @@ for ETH in /sys/class/net/*; do
    COST=0
    ZTYPE=1
    isUSB=$(readlink "$ETH" | grep -Eo '/usb[0-9]/' || true)
-   if [ "$isUSB" != "" ]
+   isCAN=$(cat "$ETH"/type)
+   if [ "$isUSB" != "" ] || [ "$isCAN" = "280" ]
```

Comment:
What is the meaning of the magic 280? Can you add a comment about it?

And wouldn't it make sense to set isCAN=(cat ETH/type = 280)

Reply:
@eriknordmark , it's just the constant defined for CAN devices by the kernel: https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/if_arp.h#L56

I'm not sure I got your second comment, are you talking about do something like the following? 
```sh
isCAN=$([ `cat "$ETH"/type` = 280 ])
``` 
I just followed the same logic for checking USB....

Reply:
isUSB is effectively a boolean, but isCAN is not. So it is harder to understand.

Reply:
I agree this name is a bit unfortunate, but isCAN is just like isUSB.... for isUSB it reads the symbolic link of the device and greps for `/usb[0-9]/` printing the matched string (if it matches), so the result is effectively a string, as the type of isCAN. Nevertheless I've changed the logic to be more similar, I think now it's better...